CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # `dev-master`
 
+* [#541](https://github.com/atoum/atoum/pull/541) New `toArray` method on the `phpString` and `object` asserters to cast value to an array ([@jubianchi])
+* [#541](https://github.com/atoum/atoum/pull/541) New `iterator` asserter ([@jubianchi])
+* [#541](https://github.com/atoum/atoum/pull/541) New `castToArray` asserter ([@jubianchi])
```

Comment:
I think it the PR description should be self-sufficient, and the changelog should only contain an unique link to the PR